conductr_cli/test/test_sandbox_features.py

```diff
@@ -1,7 +1,7 @@
 from unittest import TestCase
 from unittest.mock import call, patch, MagicMock
 from conductr_cli.sandbox_features import VisualizationFeature, LiteLoggingFeature,\
-    LoggingFeature, MonitoringFeature, \
+    LoggingFeature, MonitoringFeature, ProxyingFeature, \
```

Comment:
I'd suggest more coverage for `ProxyingFeature` in these areas:

- Output printout of during startup including when docker is enabled vs docker not enabled.
- Asserting the input argument and the call to `sandbox_proxy` during feature start depending of the input argument passed into `ProxyingFeature`'s `configure` method.
- Asserting the input argument and the call to `sandbox_proxy` during feature stop.